EIPS/eip-3336.md

```diff
@@ -0,0 +1,71 @@
+---
+eip: 3336
+title: Paged memory allocation for the EVM
+author: Nick Johnson (@arachnid)
+discussions-to: https://ethereum-magicians.org/t/eips-3336-and-3337-improving-the-evms-memory-model/5482
+status: Draft
+type: Standards Track
+category: Core
+created: 2021-03-06
+---
+
+## Simple Summary
+Changes the memory model for the EVM to use pagination, which permits more flexible use of memory by compilers without unduly complicating implementations.
```

Comment:
```suggestion
Changes the memory model for the EVM to use pagination.
```
The rest of the contents here are better suited for motivation (more flexibility) and rationale (less complicated than alternative proposals).